docker-compose.yml

```diff
@@ -61,6 +61,7 @@ services:
         environment:
             API_URI: http://api-gateway:8081
             APP_SECRET:
+            INFO_BAR:
```

Comment:
Any way to alias these, while allowing a `null` default? Would rather it was `BROWSER_INFO_BAR` on the host.

Reply:
something like this should work:
```INFO_BAR: ${BROWSER_INFO_BAR}```

Reply:
That ends up being an empty string...

Reply:
What would be the difference in this case? It shouldn't produce false positives, even in bash running `if [[ -z $INFO_BAR  ]]` and `if [[ -z $INFO_BAR  == ""]]` both work if `$INFO_BAR` is not defined or set to an empty string.

Reply:
https://github.com/libero/browser/blob/0857c9c46cdeec2ee0cfd81f03a4e3dbcb40a7e0/vendor-extra/LiberoPageBundle/src/EventListener/InfoBarListener.php#L31 (can check for an empty string, but that gets tiresome!).

Reply:
Refs https://github.com/libero/browser/pull/92#discussion_r298973075

Reply:
Could change these lines:
https://github.com/libero/browser/blob/0857c9c46cdeec2ee0cfd81f03a4e3dbcb40a7e0/vendor-extra/LiberoPageBundle/src/DependencyInjection/LiberoPageExtension.php#L51-L52

Although not the prettiest thing, might be better to catch it at the application level
```
(($config['info_bar']['text'] ?? null) ?: null)
```

Reply:
As we're using an environment variable, our case is always a special string name there that gets replaced with the actual value at runtime (with either a string or null).

https://symfony.com/doc/4.2/configuration/env_var_processors.html#built-in-environment-variable-processors doesn't have any magic to remove empty strings.

Reply:
Though 4.3 got an undocumented `nullable` processor (https://github.com/symfony/symfony/pull/29767) that should work.

Reply:
ah I see, that is tricky

Reply:
https://github.com/libero/browser/pull/94